android-sdk/src/test/java/android/util/Base64.java

```diff
@@ -0,0 +1,13 @@
+package android.util;
+
+public class Base64 {
```

Comment:
what is the purpose of this file?

Reply:
I added a dependency on the Android Base64 class to encode the credentials. The unit tests in HttpRequesterTest run outside the emulator though. So this class is added to mock the Base64 functionality needed for those tests to run.